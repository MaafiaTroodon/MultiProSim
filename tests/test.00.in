2 5 1
Proc1 2 1 1
SEND 102
HALT

Proc2 2 1 1
RECV 101
HALT
