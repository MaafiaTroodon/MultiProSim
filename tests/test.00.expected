[01] 00000: process 1 new
[01] 00000: process 1 ready
[01] 00000: process 1 running
[01] 00000: process 2 new
[01] 00000: process 2 ready
[01] 00001: process 1 blocked (send)
[01] 00001: process 2 running
[01] 00002: process 2 blocked (recv)
[01] 00003: process 1 finished
[01] 00003: process 2 finished
| 00003 | Proc 01.01 | Run 1, Block 0, Wait 0, Sends 1, Recvs 0
| 00003 | Proc 01.02 | Run 1, Block 0, Wait 1, Sends 0, Recvs 1
